src/tools/performance_test/perf_test.py

```diff
@@ -0,0 +1,244 @@
+
+
+def test_send_one(num_transactions, num_esc, num_exec):
+    send_cmd = build_send_one(num_transactions//num_esc)
+
+    for _ in range(num_exec):
+        threads = []
+
+        for i in range(num_esc):
+            threads.append(Process(target=send_one_from_client, args=(i, send_cmd)))
+
+        start = time()
+
+        for i in threads:
+            i.start()
+
+        for i in threads:
+            i.join()
+
+        end = time()
+        delta = end - start
+        print_results('send_one', delta, num_transactions)
+
+
+def create_token_accounts(num_accounts):
+    print('create_token_accounts')
+    os.chdir('user_0')
+
+    for i in range(num_accounts+1):
+        p = os.popen('./esc -w . --address ' + get_account_address(i) + ' > /dev/null 2>&1', 'w')
+        p.write('{"run":"get_me"}\n')
+        p.write('{"run":"create_token_account", "token":"00000000"}\n')
+        p.close()
+
+    os.chdir('..')
+
+
+def add_tokens_to_first_account():
+    os.chdir("user_0")
+
+    p = os.popen('./esc -w . > /dev/null 2>&1', 'w')
+    p.write('{"run":"get_me"}\n')
+    p.write('{"run":"add_tokens", "amount":"20000000"}\n')
+    p.close()
+
+    os.chdir("..")
+
+
+def move_tokens_to_first_accounts(num_esc):
+    os.chdir('user_0')
+
+    p = os.popen('./esc -w . > /dev/null 2>&1', 'w')
+    p.write('{"run":"get_me"}\n')
+
+    for i in range(1, num_esc):
+        p.write('{"run":"move_tokens", "token":"00000000", "to":"' + get_account_address(i) + '", "amount":"1000000"}\n')
+
+    p.close()
+    os.chdir('..')
+
+
+def build_move_tokens(txs_per_client):
+    move_cmd = []
+
+    offset = 6000
+
+    for i in range(offset, offset + txs_per_client):
+        move_cmd.append('{"run":"move_tokens", "token":"00000000", "to":"' + get_account_address(i) + '","amount":1}\n')
+
+    return move_cmd
+
+
+def move_tokens_from_client(client_id, move_cmd):
+    os.chdir('user_' + str(client_id))
+    print('move_tokens from: ' + str(os.getcwd()))
+
+    p = os.popen('./esc -w . > /dev/null 2>&1', 'w')
+    p.write('{"run":"get_me"}\n')
+
+    for cmd in move_cmd:
+        p.write(cmd)
+
+    p.close()
+
+
+def test_move_tokens(num_transactions, num_esc, num_exec):
+    move_cmd = build_move_tokens(num_transactions//num_esc)
+
+    for _ in range(num_exec):
+        threads = []
+        for i in range(num_esc):
+            threads.append(Process(target=move_tokens_from_client, args=(i, move_cmd)))
+
+        start = time()
+
+        for i in threads:
+            i.start()
+
+        for i in threads:
+            i.join()
+
+        end = time()
+        delta = end - start
+        print_results('move_tokens', delta, num_transactions)
+
+
+def run_tests():
+    num_accounts = 10000
+    num_transactions = 100000
+    num_esc = 16
+
+    prepare_accounts(num_accounts, num_esc)
+    test_get_me(num_transactions, num_esc, num_exec=1)
+    test_send_one(num_transactions, num_esc, num_exec=10)
+
+    # prepare_token_accounts(num_accounts, num_esc)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS125)